ci/browser_tests.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-
-SAUCECONNECT_URL="https://saucelabs.com/downloads/sc-4.3.16-linux.tar.gz"
+SC_VERSION="4.4.0"
```

Comment:
Can you split out the changes to `browser_tests.sh` into another PR and merge that?